types/api/v1beta1/event_data.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/wrappers.proto";
+
+message EventValue {
+    oneof value {
+        google.protobuf.Int32Value int32 = 1; // intT
+        google.protobuf.UInt32Value uint32 = 2; // uintT
+        google.protobuf.Int64Value int64 = 3; // longT
+        google.protobuf.UInt64Value uint64 = 4; // ulongT
+        google.protobuf.UInt64Value off = 5; // offT
+        google.protobuf.UInt32Value mode = 6; // modeT
+        google.protobuf.UInt32Value dev = 7; // devT
+        google.protobuf.UInt64Value size = 8; // sizeT
+        google.protobuf.UInt64Value pointer = 9; // pointerT
+        google.protobuf.StringValue string = 10; // strT
+        StringArrayValue string_array = 11; // strArrT
+        SockAddrValue sockaddr = 12; // sockaddrT
+        google.protobuf.BytesValue bytes = 13; // bytesT
+        UInt16Value uint16 = 14; // u16T
+        CredValue cred = 15; // credT
+        IntArrayValue int_array = 16; // intArr2T
+        UInt64ArrayValue uint64_array = 17; // uint64ArrT
+        UInt8Value uint8 = 18; // u8T
+        TimespecValue timespec = 19; // timespecT
+        StringArrayValue args_array = 20; // argsArrT
+        google.protobuf.BoolValue bool = 21; // boolT
+    }
+}
+
+message StringArrayValue {
+    repeated google.protobuf.StringValue array = 1;
+}
+
+message IntArrayValue {
+    repeated google.protobuf.Int32Value array = 1;
+}
+	
+message UInt64ArrayValue {
+    repeated google.protobuf.UInt64Value array = 1;   
+}
+	
+// TODO: what do you think we don't have a uint16 type in protobuf,
+// so we have a message to inform the size, so the user converts the internal value
```

Comment:
What is "a message to inform the size"?

Reply:
`message Uint16Value` -> a message specific for the type, though the user will have to convert because we don't have uint16 support in protobuf. Makes sense?